### ISSUE TITLE
allow iobox to continue in some failure cases

### ISSUE THREAD

**Issue by hongsudt** [informatics-isi-edu/iobox-win32#30]
We discussed implementing: 

```
continueAfter = [ <error cases>]
where <error cases> is one of 
- http error code such as 409 (for conflict)
- "ZERO_RESULT": for a get request with 0 row in the result
- '*": for any error cases 
```

**Comment by svoinea** [comment]
Implemented through the commit 6bcf796519a0543fd9d4cfe68dbdc78cc69ee4c3
Updated also the README file with the **ermrest** `continueAfter` option.

**Comment by hongsudt** [comment]
I don't have a way to test this, so unless anyone else can help review this, I will close the issue.
